yaml/Get-HelpPreview.yml

```diff
@@ -0,0 +1,78 @@
+﻿examples:
```

Comment:
No need to include this `yaml` folder into PR. We only should keep the sources under the source control. These files are purely the artifacts from the markdown, similar to maml xmls.